central/processbaseline/datastore/datastore.go

```diff
@@ -26,8 +26,8 @@ type DataStore interface {
 	RemoveProcessBaseline(ctx context.Context, key *storage.ProcessBaselineKey) error
 	RemoveProcessBaselinesByDeployment(ctx context.Context, deploymentID string) error
 	RemoveProcessBaselinesByIDs(ctx context.Context, ids []string) error
-	UpdateProcessBaselineElements(ctx context.Context, key *storage.ProcessBaselineKey, addElements []*storage.BaselineItem, removeElements []*storage.BaselineItem, auto bool) (*storage.ProcessBaseline, error)
-	UpsertProcessBaseline(ctx context.Context, key *storage.ProcessBaselineKey, addElements []*storage.BaselineItem, auto bool, lock bool) (*storage.ProcessBaseline, error)
+	UpdateProcessBaselineElements(ctx context.Context, key *storage.ProcessBaselineKey, addElements []*storage.BaselineItem, removeElements []*storage.BaselineItem, auto bool, locked bool) (*storage.ProcessBaseline, error)
+	UpsertProcessBaseline(ctx context.Context, key *storage.ProcessBaselineKey, addElements []*storage.BaselineItem, auto bool, lock bool, userLock bool) (*storage.ProcessBaseline, error)
```

Comment:
nit:  one has name `locked` the other `userLock`.  Are they both representing the same thing?  If so, please name them the same so it is clear.